.env

```diff
@@ -0,0 +1,15 @@
+DB_USER=postgres
+DB_PASSWORD=123456
+DB_DATABASE=ah_node
+DB_HOST=localhost
+
+TEST_DB_USER=
+TEST_DB_PASSWORD=
+TEST_DB_DATABASE=
+TEST_DB_HOST=
+
+PROD_DB_USER=
+PROD_DB_PASSWORD=
+PROD_DB_DATABASE=
+PROD_DB_HOST=
+
```

Comment:
Are you supposed to pass the `.env` file to github?

Reply:
No, it was a mistake, it came on after I did the rebase, I didn't spot it.

Reply:
@veeqtor rebasing shouldn't cause this. Did you get a conflict after you rebased?